models/userDao.js

```diff
@@ -51,13 +51,71 @@ const getPlacesInLibrary = async (userId, libraryId) => {
     throw error;
   }
 };
+
+const createLibrary = async (userId, libraryName) => {
+  try {
+    return await appDataSource.query(
+      `
+			INSERT INTO libraries (user_id, name)
+			VALUES(?, ?)
+			`,
+      [userId, libraryName]
+    );
+  } catch (err) {
+    const error = new Error("INVALID_DATA_INPUT");
+    error.statusCode = 400;
+    throw error;
+  }
+};
+
+const createPlaceLike = async (userId, libraryId, placeId) => {
+  const queryRunner = appDataSource.createQueryRunner();
+
+  await queryRunner.connect();
+  await queryRunner.startTransaction();
+
+  try {
+    const checkOwner = await queryRunner.query(
+      `SELECT * FROM libraries
+      WHERE id = ? AND user_id = ?
+      `,
+      [libraryId, userId]
+    );
+    if (checkOwner[0].id === undefined) {
+      throw new Error("NOT_YOUR_LIBRARY");
+    }
```

Comment:
아래 내용처럼 바꾸어보는 것은 어떨까요?
권한 없음의 상태코드를 부여하는 내용과 가독성을 위해 ===undefined 내용을 삭제하는 기획이 반영된 내용입니다.

```suggestion
    const [ checkOwner ] = await queryRunner.query(
      `SELECT * FROM libraries
      WHERE id = ? AND user_id = ?
      `,
      [libraryId, userId]
    );
    if (!checkOwner) {
      const error = new Error("NOT_YOUR_LIBRARY");
      error.statusCode = 401;
      throw error;
    }
```